server/game/cards/04-MM/MatterMaker.js

```diff
@@ -0,0 +1,17 @@
+const Card = require('../../Card.js');
+
+class MatterMaker extends Card {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            targetController: 'current',
+            effect: ability.effects.canPlayNonHouse(({
```

Comment:
I think you just want canPlay here

Reply:
Nice catch. There was actually a bigger problem here. I think the old code was letting ANY card be played out of house. I've updated to use ```canPlay``` and added a test case to check for not being able to play creatures and actions. Fixed in update.